features/src/main/feature/camel-features.xml

```diff
@@ -2332,8 +2332,8 @@
     </feature>
     <feature name='camel-swift' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
-        <bundle dependency='true'>wrap:mvn:com.prowidesoftware/pw-swift-core/SRU2023-10.1.13</bundle>
-        <bundle dependency='true'>wrap:mvn:com.prowidesoftware/pw-iso20022/${prowide-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.prowidesoftware/pw-swift-core/SRU2023-10.1.13$Bundle-Version=0.0.0.SRU2023-10_1_13</bundle>
+        <bundle dependency='true'>wrap:mvn:com.prowidesoftware/pw-iso20022/${prowide-version}$Bundle-Version=0.0.0.SRU2023-10_1_4</bundle>
```

Comment:
Use the maven property to set the version

Reply:
The version coming from the version placeholder doesn't work
`<prowide-version>SRU2023-10.1.4</prowide-version>`
```
karaf@root()> feature:install camel-swift
org.osgi.framework.BundleException: Could not create bundle object.
        at org.apache.felix.framework.Felix.installBundle(Felix.java:3339)
        at org.apache.felix.framework.BundleContextImpl.installBundle(BundleContextImpl.java:147)
        at org.eclipse.equinox.internal.region.BundleIdBasedRegion.installBundle0(BundleIdBasedRegion.java:120)
        at org.eclipse.equinox.internal.region.BundleIdBasedRegion.installBundleAtLocation(BundleIdBasedRegion.java:100)
        at org.apache.karaf.features.internal.service.BundleInstallSupportImpl.installBundle(BundleInstallSupportImpl.java:135)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.installBundle(FeaturesServiceImpl.java:1145)
        at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:906)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1069)
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.lambda$doProvisionInThread$13(FeaturesServiceImpl.java:1004)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: invalid version "SRU2023-10.1.4": non-numeric "SRU2023-10"
        at org.osgi.framework.Version.parseInt(Version.java:169)
        at org.osgi.framework.Version.<init>(Version.java:126)
        at org.osgi.framework.Version.valueOf(Version.java:255)
        at org.osgi.framework.Version.parseVersion(Version.java:226)
        at org.apache.felix.framework.util.manifestparser.ManifestParser.<init>(ManifestParser.java:129)
        at org.apache.felix.framework.BundleRevisionImpl.<init>(BundleRevisionImpl.java:117)
        at org.apache.felix.framework.BundleImpl.createRevision(BundleImpl.java:1290)
        at org.apache.felix.framework.BundleImpl.<init>(BundleImpl.java:114)
        at org.apache.felix.framework.Felix.installBundle(Felix.java:3287)
        ... 12 more
Caused by: java.lang.NumberFormatException: For input string: "SRU2023-10"
        at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
        at java.base/java.lang.Integer.parseInt(Integer.java:668)
        at java.base/java.lang.Integer.parseInt(Integer.java:786)
        at org.osgi.framework.Version.parseInt(Version.java:167)
        ... 20 more
```